config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml

```diff
@@ -13,4 +13,4 @@ spec:
   pullpolicy: Always
   pullsecret: multiclusterhub-operator-pull-secret
   searchimageoverrides:
-    redisgraph_tls: quay.io/open-cluster-management/redisgraph-tls:2.2.0-PR40-82f419595a22d6f15538ae2379b3f286e0273539
+    redisgraph_tls: quay.io/stolostron/redisgraph-tls:2.2.0-PR40-82f419595a22d6f15538ae2379b3f286e0273539
```

Comment:
This image tag doesn't exist in the new org.  Just checking that we don't use it.  Maybe change it to a dummy value if it's only a sample.

Reply:
I believe this is why Travis failed.

Reply:
Updated to latest tag in stolostron.